cmake/versions.json

```diff
@@ -5,7 +5,7 @@
       "git_url" : "https://github.com/nv-legate/legate.core.git",
       "git_shallow": false,
       "always_download": false,
-      "git_tag" : "a405f595603238c8557cb5fefd3981d190a2fb1d"
+      "git_tag" : "4b79075eb5d7035d501c334c87a87939af79abc2"
```

Comment:
Is there something in legate that needs to be updated for this?

Reply:
It looks like we do need a later version of legate.core than what is currently specified in versions.json.: https://github.com/nv-legate/cunumeric/pull/993/checks#step:7:50. To be more specific we need the file `legate/continuous_integration/build-docker-image`. 

The selected SHA does not directly point to the specific change which introduced the aforementioned dependency but it advances legate.core to the point where we know we have a successful build using build-docker-image.

Reply:
Yup, that makes sense. Please proceed.